src/rebar_utils.erl

```diff
@@ -289,10 +291,52 @@ line_count(PatchLines) ->
     Tokenized = string:tokens(PatchLines, "\n"),
     {ok, length(Tokenized)}.
 
+check_min_otp_version(undefined) ->
+    ok;
+check_min_otp_version(MinOtpVersion) ->
+    %% Fully-qualify with ?MODULE so the function can be meck'd in rebar_utils_SUITE
+    OtpRelease = ?MODULE:otp_release(),
+    {MinMajor, MinMinor} = split_version(MinOtpVersion),
+    {OtpMajor, OtpMinor} = split_version(OtpRelease),
```

Comment:
This will fail when the `OTP_VERSION` file cannot be found, or on a major version that only returns "17" for example.